src/main/java/com/greenfoxacademy/goddesstribesbackend/models/MockData.java

```diff
@@ -0,0 +1,42 @@
+package com.greenfoxacademy.goddesstribesbackend.models;
+
+import com.greenfoxacademy.goddesstribesbackend.models.dtos.*;
+
+import java.time.LocalDateTime;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+public class MockData {
+
+  public static BuildingDTO farm = new BuildingDTO(2L, BuldingTypeENUM.FARM, 1,
```

Comment:
1.- is there a reason we start with 2L
2.- is there a reason for using the L markup? Is it necessary? (it might be)

Reply:
unfortunately, there is.